transmogrifier/sources/xml/ead.py

```diff
@@ -461,35 +463,18 @@ def _parse_date_elements(
         """
         dates = []
         for date_element in collection_description_did.find_all("unitdate", normal=True):
-            date_value = date_element.get("normal", "").strip()
-            if date_value == "":
+            date_string = date_element.get("normal", "").strip()
+
+            if date_string == "":
                 continue
 
             date_instance = timdex.Date()
 
             # get valid date ranges or dates
-            if "/" in date_value:
-                gte_date, lte_date = date_value.split("/")
-                if gte_date != lte_date:
-                    if validate_date_range(
-                        gte_date,
-                        lte_date,
-                        source_record_id,
-                    ):
-                        date_instance.range = timdex.DateRange(
-                            gte=gte_date,
-                            lte=lte_date,
-                        )
-                else:
-                    # get valid date (if dates in ranges are the same)
-                    date_string = gte_date
-                    if validate_date(
-                        date_string,
-                        source_record_id,
-                    ):
-                        date_instance.value = date_string
-            elif validate_date(date_value, source_record_id):
-                date_instance.value = date_value
+            if "/" in date_string:
+                date_instance = cls._parse_date_range(date_string, source_record_id)
```

Comment:
I think this is a nice change.  I almost commented in the previous PR that this field method was a bit unwieldy, but struggling a bit with how much of that refactoring -- _particularly_ around dates -- to suggest in this first pass of just getting to a field method approach.

Either way, much easier to scan and understand now.

Reply:
Agreed, great work!